account-keys/slip10/src/mnemonic2slip.py

```diff
@@ -7,17 +7,23 @@
 # our own pair of HKDF instances to construct both view and spend keys.
 #
 # Much of this code was copy/pasted from
-# https://github.com/satoshilabs/slips/slip-0010/testvectors.py and probably
-# should not be re-used in a production setting.
+# https://raw.githubusercontent.com/satoshilabs/slips/master/slip-0010/testvectors.py
+# and probably should not be re-used in a production setting.
 #
 
 import hashlib
+import struct
+
+import ed25519
 import hkdf
 import hmac
 import mnemonic
 import sys
 
 
+privdev = 0x80000000
```

Comment:
`privdev` stands for what?

Reply:
I assume "privkey-only derivation" or something like that---it's literally the difference between "hardened" and "unhardened" in the BIP32 path...

Reply:
Maybe `hardening_factor` would be easier to read. Small detail ---

Reply:
Yeah, I'm going to leave it, just because that's what the slip10 `testvectors.py` (also now included in this PR) calls it.